docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+
+## The Different Pelican URL Schemes
+When running client commands with a request URL, there are three different URL schemes a user can use: `pelican`, `osdf`, and `stash`. This page uses examples with the `pelican://` URL scheme. Below is a description of the three URL schemes and what to expect when they are used:
+
+### The `pelican://` URL Scheme:
+When using the `pelican://` scheme, pelican does not know what federation to communicate with and therefore, the user must specify their federation URL either as the host-name in the URL itself or within the `-f` flag. The examples below show how you can format your request URLs in both of these ways.
+
+>**Note:** When you set a federation discovery URL within a `pelican://` URL, within configuration files, or with the `-f` flag, do not expect the client to work if the discovery URL contains a path (e.g. `osg-htc.org/PathToMyFederation`). The client will assume that the path provided is the namespace prefix for the object which will end in failure.
+
+### The `osdf:///` URL Scheme:
+When using the `osdf:///` scheme, pelican will load in the federation metadata automatically with the Open Science Data Federation (OSDF)  defaults. This scheme should only be used if your federation or data can be accessed through the OSDF. When using this scheme, you should not specify the federation URL at all, as Pelican will handle this automatically. For example:
+```bash
+pelican object get osdf:///<namespace-prefix></path/to/file> <local/path/to/file>
+```
+>This scheme has three slashes (`///`) after the `osdf` because the hostname is left empty to be automatically populated therefore, just start the URL with the namespace prefix. Pelican also supports the `osdf://` scheme (with two slashes) in case the user forgets to pass the third slash, but this is not recommended.
+
+### The `stash:///` URL Scheme:
+This URL scheme comes from legacy code called `stashcp` that Pelican was built off of and is still accepted for backwards compatibility. This URL scheme functions identically to the `osdf:///` URL scheme, loading in OSDF defaults. Here is an example:
+```bash
+pelican object get stash:///<namespace-prefix></path/to/file> <local/path/to/file>
+```
+
+## Get a Public Object from your Federation
+
+To use the pelican client to get public objects from a federation, use Pelican's `object get` sub-command
+
+### Getting an Object Using the pelican:// URL Scheme:
+To use the `pelican://` URL scheme, you need to specify the federation URL within the request URL. To do this, you would format the `object get` command like so:
+```bash
+pelican object get pelican://<federation-url></namespace-prefix></path/to/file> <local/path/to/file>
+```
+Or, you can use the `-f` flag with the command, and format the request URL like so:
+```bash
+pelican object get -f <federation-url> pelican://</namespace-prefix></path/to/file> </local/path/to/file>
```

Comment:
I don't like the idea of presenting this as an option to users. Can we say you can either do:
```

pelican object get -f <fed url> /namespace/prefix/object /local/path/to/file
```

OR

```
pelican object get pelican://<fed url>/namespace/prefix/object /local/path/to/file
```

To the extent possible, let's try to make sure users think of pelican-schemed URLs as a way to provide the federation discovery URL and object name in a single vessel, not to split things up the confusing way we do with `osdf:///`

Reply:
I think I am just going to remove the examples with `-f` and just leave it as a note. I was not too fond of having it either, takes up space and can be confusing/not useful to mention